corehq/apps/export/views.py

```diff
@@ -973,7 +975,20 @@ def dispatch(self, request, *args, **kwargs):
         if not (self.has_edit_permissions or self.has_view_permissions
                 or (self.is_deid and self.has_deid_view_permissions)):
             raise Http404()
-        return super(BaseExportListView, self).dispatch(request, *args, **kwargs)
+
+        self.request = request
+
+        if domain_has_daily_saved_export_access(self.domain) or domain_has_excel_dashboard_access(self.domain):
+            self.set_notify_new_daily_saved_export()
+
+        return super(BaseExportListView, self).dispatch(self.request, *args, **kwargs)
+
+    def set_notify_new_daily_saved_export(self):
+        self.request.notify_new_daily_saved_export = False
+
+        if self.request.session.get('notify_new_daily_saved_export', True):
+            self.request.notify_new_daily_saved_export = True
+            self.request.session['notify_new_daily_saved_export'] = False
```

Comment:
I don't think it's a great user experience for the modal to show on every new session. I think it should be a one-time thing. I know it's a bit more work, but I think the "read" status of the notification should reside somewhere in permanent storage. You could look at the [`corehq.apps.notifications.models.Notification`](https://github.com/dimagi/commcare-hq/blob/2d42bbf3088f7ac1ec7a881df1f7f4d27ca5909c/corehq/apps/notifications/models.py#L20) model might provide some inspiration.